src/playlist-maker.js

```diff
@@ -3,6 +3,16 @@ var setupAutoadvance = require('./autoadvance.js');
 var isArray = Array.isArray || function(array) {
   return Object.prototype.toString.call(array) === '[object Array]';
 };
+var isInSources = function(arr, src) {
+  var i = 0;
+  for (; i < arr.length; i++) {
+    if (arr[i] === src || arr.src === src) {
```

Comment:
Though, it's weird that my tests are passing. Though, maybe it's because my tests only exercise one part of the OR.